product.json

```diff
@@ -57,6 +57,6 @@
 		"msrvida.azdata-sanddance"
 	],
 	"extensionsGallery": {
-		"serviceUrl": "https://sqlopsextensions.blob.core.windows.net/marketplace/v1/extensionsGallery.json"
+		"serviceUrl": "https://sqlopsextensions.blob.core.windows.net/marketplace/v1/extensionsGallery-test.json"
```

Comment:
You can't commit this in, please undo (it's only for test-on-your-machine purposes)